R/teal_slices.R

```diff
@@ -103,11 +103,7 @@ teal_filters <- function(...,
 deep_copy_filter <- function(filter) {
   shiny::isolate({
     filter_copy <- lapply(filter, function(slice) {
-      if (inherits(slice, "teal_slice_expr")) {
-        do.call(teal.slice::filter_expr, args = reactiveValuesToList(slice))
-      } else {
-        do.call(teal.slice::filter_var, args = reactiveValuesToList(slice))
-      }
+      do.call(teal.slice::teal_slice, args = reactiveValuesToList(slice))
```

Comment:
This could use `as.teal_slice` if we export it or steal it from `teal.slice`.

Reply:
Yes, it could use `as.list.teal_slice`.

Reply:
That's what I mean, sorry. `as.list.teal_slice` is exported already, as I recall.

Reply:
changed to `as.teal_slice(as.list(slice))`